libs/content/workspace-ffi/SwiftWorkspace/Sources/Editor/iOSMTK.swift

```diff
@@ -163,39 +163,34 @@ public class iOSMTKTextInputWrapper: UIView, UITextInput, UIDropInteractionDeleg
             self.mtkView.setNeedsDisplay()
         }
     }
+    
+    func sendImage(img: Data, isPaste: Bool) {
+        let imgPtr = img.withUnsafeBytes { (pointer: UnsafeRawBufferPointer) -> UnsafePointer<UInt8> in
+            return pointer.baseAddress!.assumingMemoryBound(to: UInt8.self)
```

Comment:
how is this returning something if this function has no return type 😵‍💫 swift is so confusing to me

Reply:
that return value is going into the imgPtr I think? `->` is an inline function

Reply:
Yeah its a little bit cryptic. The inner closure gives the outer closure its type.